src/components/organisms/Notification/NotificationHeader/index.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import { MemoryHistory } from 'history';
 import { RouteComponentProps, withRouter } from 'react-router';
-import Logo from 'components/atoms/icons/LogoDismoi';
-import { useFacetName } from 'libs/facets/useFacetName.hook';
+import WrappedLogoDismoi from 'components/atoms/icons/LogoDismoi';
+import WrappedLogoLMEL from 'components/atoms/icons/LogoLMEL';
```

Comment:
> Following previous comment...

...or better: have a `logo` property in the theme instead.
Nothing forbid us to have a reference to a component in the `theme` object!

Reply:
Please, do abuse of the theme system!